_data/engine-cli/docker_attach.yaml

```diff
@@ -1,10 +1,18 @@
 command: docker attach
-short: Attach to a running container
+short: Attach local standard input, output, and error streams to a running container
 long: |-
-  Use `docker attach` to attach to a running container using the container's ID
-  or name, either to view its ongoing output or to control it interactively.
+  Use `docker attach` to attach your terminal's standard input, output, and error
+  (or any combination of the three) to a running container using the container's
+  ID or name. This allows you to view its ongoing output or to control it
+  interactively, as though the commands were running directly in your terminal.
+
+  > **Note:**
+  > The `attach` command will display the output of the `ENTRYPOINT/CMD` process.  This
+  > can appear as if the attach command is hung when in fact the process may simply
+  > not be interacting with the terminal at that time.
+
   You can attach to the same contained process multiple times simultaneously,
-  screen sharing style, or quickly view the progress of your detached process.
+  even as a different user with the appropriate permissions.
```

Comment:
Does this mean as a different user on the host? Might want to clarify that part.